.github/workflows/publish-and-use.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Publish & use NPM package"
```

Comment:
Here is an example of the publish and use action passing in a github action on the developers repo - https://github.com/bojinovic/filecoin-solidity-library/actions/runs/9175094751 - we need to confirm that this works on this repo after we merge the PR